### ISSUE TITLE
`aria-label` missing for top menu

### ISSUE THREAD

**Issue by ernilambar** [WordPress/twentyseventeen#79]
In `navigation-top.php`, `aria-label` missing for top menu.

It could be like this.
`<nav id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php _e( 'Top Menu', 'twentyseventeen' ); ?>">`
